source/testnet/guides/become-baker.rst

```diff
@@ -207,16 +193,18 @@ user first needs to shut down the current running node (either by pressing
 ``Ctrl + C`` on the terminal where the node is running or using the
 ``concordium-node-stop`` executable).
 
-After placing the file in the appropriate directory, start the
-node again using ``concordium-node``. The node will automatically start baking
-when the baker is included in the bakers for the current epoch. This will happen
-when finishing the epoch after the one in which the transaction for adding the
-baker was finalized.
+After placing the file in the appropriate directory (already done in the
+previous command when specifying the output file), start the node again using
+``concordium-node``. The node will automatically start baking when the baker
+gets included in the bakers for the current epoch. This change will be executed
+immediately and will take effect when finishing the epoch after the one in which
+the transaction for adding the baker was finalized.
 
 .. note::
 
    If the transaction for adding the baker was finalized during epoch `E`, the
-   baker will be active when epoch `E+2` starts.
+   baker will be considered as being part of the baking committee when epoch
+   `E+2` starts.
```

Comment:
You should remove all mentions of finalization as it pertains to epochs and stake changes. It is not relevant for the timing and is likely to confuse.

Reply:
References to finalization have been removed (except in the Finalization section).